.gitignore

```diff
@@ -119,3 +119,4 @@ doc/build/html/index.html
 doc/tmp.sv
 env/
 doc/source/savefig/
+virtualenvs/
```

Comment:
revert this addition to the gitignore. it is effective only for *your* environment.